Models/biophysics_calibration/pyabc_cellmodeller.py

```diff
@@ -55,42 +101,73 @@ def sim_time_parameters(file_path):
     Must be written as dt = X.Y in the module file
     
     @param  file_path   str of file name containing parameters to be modified
+    @return dt          Simulation time step specified in CellModeller module file
     """
     search_dt = f'dt = (\d+\.\d+)'
     with open(file_path, "r+") as file:
         file_contents = file.read()
         dt = re.findall(search_dt, file_contents)
     return float(dt[0])
+       
+def distance_calculation(sim_stats, exp_stats):
+    """
+    Calculate Euclidean distance between simulations and experimental observations
+    
+    @param  sim_stats   Dict of simulated summary statistics
+    @param  exp_stats   Dict of experimental summary statistics
+    @return distance    Euclidean distance between simuluations and experiment
+    """
+    distance_list = []
+    for x in sim_stats.keys():
+        diff = np.abs(sim_stats[x] - exp_stats[x])
+        distance_list.append(diff)
+
+    distance = np.linalg.norm(distance_list)
+
+    return distance    
     
 if __name__ == '__main__':
+    # Define ABC-SMC settings
+    n_cores = 8
+    population_size = 10
+    min_epsilon = 0.05
+    max_populations = 10
+
     # Define experimental data
-    sumstat1 = 10
-    sumstat2 = 0.1
-    observation = {"data": np.array([sumstat1, sumstat2])} # currently using dummy values
+    exp_summary_stats = {}
+    exp_summary_stats['growth_rate_vs_centroid'] = 0.05
```

Comment:
I believe we need a generalized code that everyone can use so that we shouldn’t assign a value for summary stats (running on experimental data) in the ABC  code. On the other hand, running summary stats on experimental data is normally done just once. So, you can run summary stats for experimental data within your ABC code and fetch their values.

Reply:
I interpreted this as place-holder code to do just that.